SwiftGame.xcworkspace/xcshareddata/IDETemplateMacros.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>FILEHEADER </key>
+	<string>//
+// ___PACKAGENAME___
+// Created by: ___FULLUSERNAME___ and Pedro Ferrari on ___DATE___
+// Copyright (c) 2018 ThePugas
+//
+//  _________________________________________
+// |   _____ _       _____                   |
+// |  |_   _| |_ ___|  _  |_ _ ___ ___ ___   |
+// |    | | |   | -_|   __| | | . | .&apos;|_ -|  |
```

Comment:
Check if &apos; is a safe character or it does produces errors when automating the header creation on new files.

Reply:
New headers are being created correctly